app/data/census_household.json

```diff
@@ -893,15 +910,16 @@
                 {
                     "type": "interstitial",
                     "id": "household-member-begin",
-                    "title": "{{ answers.person_full_name[group_instance] }}",
+                    "title": "household member begin",
                     "sections": [
                         {
-                            "title": "{{ answers.person_full_name[group_instance] }}",
+                            "title": "{{ [answers.first_name[group_instance]|default('', true), answers.last_name[group_instance]|default('', true)]|join(' ') }}",
                             "id": "household-member-begin-section",
                             "questions": [
                                 {
                                     "id": "household-member-begin-question",
                                     "title": "1. The following questions are for:",
+                                    "description": "<div style='background-color: #efefef; padding: .7em; display: inline-block; border: 2px solid #ccc; font-weight: bold;'>{{ [answers.first_name[group_instance]|default('', true), answers.middle_names[group_instance]|default('', true), answers.last_name[group_instance]|default('', true)]|join(' ') }}</div>",
```

Comment:
🎨  Frontend support required to help remove the inline CSS here to somewhere sensible. I think this is the same style that is required to display household names on the household composition summary screen, so maybe the styles can be re-used.